src/server/rpc/procedures/hawaii-co2-data/hawaii-co2-data.js

```diff
@@ -0,0 +1,103 @@
+const lineReader = require('line-reader');
+const path = require('path');
+
+const table = [
+    ['Year', 'Month', 'Interpolated ppm', 'Seasonally adjusted ppm']
+];
+const table1 = [
+    ['Year', 'Month', 'Interpolated ppm']
+];
+const table2 = [
+    ['Year', 'Month', 'Seasonally adjusted ppm']
+];
+
+
+lineReader.eachLine(path.join(__dirname,'co2_mm_mlo.txt'), function (line) {
+    if (!line.startsWith('#')) {
+        let year = line.substring(0, 4);
+        let month = line.substring(4, 8).trim();
+        let inter = line.substring(38, 44);
```

Comment:
do we know what spec does the file follow? usually the values are separated by a common separator and in that case, it's best to use that separator to access each value. 
one of the issues with using absolute sting index for grabbing the data is what if the number is bigger than what you anticipate?

Reply:
The database always follows the same format throughout: https://www.nhc.noaa.gov/data/hurdat/hurdat2-format-atlantic.pdf

Reply:
Wait, the above link is to the hurricane dataset. Am I missing something?

Reply:
Oh sorry, it seems that I linked the wrong file in that comment. GitHub won't let me link the correct file here, so is there an email address I can send the file to for you to access?

Reply:
Is there a link you could post? If not, my email address is brian.broll@gmail.com

It is pretty important that we can reference the source for the data files for the NetsBlox users, too, so we should figure out how we can give proper credit in a service annotation.